styles/colors.css

```diff
@@ -1,18 +1,96 @@
 :root {
-  --bs-danger-rgb: var(--stanford-digital-red-dark-rgb);
-  --bs-danger-bg-subtle: rgba(var(--bs-danger-rgb), 10%);
-  --bs-danger-border-subtle: rgb(var(--bs-danger-rgb));
-  --bs-danger-text-emphasis: rgb(var(--bs-danger-rgb));
+  --bs-blue: var(--stanford-sky-dark);
+  --bs-indigo: #6610f2;
```

Comment:
I'd really prefer we only pull in the colors we need to do the functions in `colors/text.html`.   Let's keep this to just what we need as it makes maintenance easier.

Reply:
How do we know what colors we'll need? My hope is we don't have to think about it, and if we ask for a bootstrap color in our applications we get one of the branding colors.